### ISSUE TITLE
Add references provided by reviewer

### ISSUE THREAD

**Issue by EmilHvitfeldt** [EmilHvitfeldt/smltar#107]
Jeffrey L Elman. 1990. Finding structure in time. Cognitive science, 14(2):179–211.
Sepp Hochreiter and J ̈urgen Schmidhuber. 1997. Long short-term memory. Neural computation,
9(8):1735–1780.

Yoon Kim. 2014. Convolutional neural networks for sentence classification. In Proceedings of
the 2014 Conference on Empirical Methods in Natural Language Processing (EMNLP), pages
1746–1751, Doha, Qatar. Association for Computational Linguistics.

Diederik P Kingma and Jimmy Ba. 2014. Adam: A method for stochastic optimization. arXiv
preprint arXiv:1412.6980.

Nitish Srivastava, Geoffrey Hinton, Alex Krizhevsky, Ilya Sutskever, and Ruslan Salakhutdinov.
2014. Dropout: a simple way to prevent neural networks from overfitting. The journal of
machine learning research, 15(1):1929–1958.
